src/components/SoqlListView.component

```diff
@@ -29,148 +29,195 @@
+                    <th scope="col" class="slds-cell-shrink">
+                        <input type="checkbox" name="cbxContactAll" id="cbxAll" onchange="CheckAllSLVObj(this.checked);" />
+                        <span class="slds-assistive-text">Select All</span>
+                    </th>
+                    <th scope="col" class="slds-is-sortable">
+                        <label class="">
+                            <apex:commandLink action="{!SortItem}" rerender="panelData" status="statusLoad" value="{!$ObjectType.Contact.Fields[pageController.strNameField].Label}" >
+                                <apex:param assignTo="{!SortItemField}" name="SortItemField" value="{!pageController.strNameField}" />
+                                <apex:outputText style="font-size:1.0em;font-weight:bold;font-family:symbol" 
+                                    value="{!IF(SortItemField=pageController.strNameField, IF(SortItemAsc, ' ↑' , ' ↓'), '')}" escape="false" />
+                            </apex:commandLink>
+                        </label>
+                        <span class="slds-assistive-text">Sort</span>
+                    </th>
+                    <apex:repeat value="{!pageController.strFields}" var="f" > 
+                        <apex:outputText rendered="{!f != pageController.strNameField}" >
+                            <th style="max-width: 10%" class="slds-truncate">
+                                <apex:outputText value="{!$ObjectType.Contact.Fields[f].Label}" rendered="{!$ObjectType.Contact.Fields[f].Type == 'multipicklist'}" />
+                                <apex:commandLink action="{!SortItem}" rerender="panelData" status="statusLoad" value="{!$ObjectType.Contact.Fields[f].Label}" 
+                                    rendered="{!$ObjectType.Contact.Fields[f].Type != 'multipicklist'}" >
+                                    <apex:param assignTo="{!SortItemField}" name="SortItemField" value="{!f}" />
+                                    <apex:outputText style="font-size:1.0em;font-weight:bold;font-family:symbol" 
+                                        value="{!IF(SortItemField=f, IF(SortItemAsc, ' ↑' , ' ↓'), '')}" escape="false" />
+                                </apex:commandLink>
+                            </th>
+                        </apex:outputText> 
+                    </apex:repeat>         
+                </tr>
+            </thead>
+            <tbody>
+                <apex:repeat value="{!listSLVObj}" var="item" rows="{!cRecordsPerPage}" >
+                    <tr class="slds-cell-shrink">
+                        <td>
+                            <apex:inputCheckbox value="{!item.checked}" styleClass="cbxSLVObj" />
+                        </td>
+                        <td class="slds-text-body--small">
+                            <apex:outputLink value="/{!item.sobj['id']}" >
+                                <apex:outputField value="{!item.sobj[pageController.strNameField]}" />
```

Comment:
dates are no longer showing up like they used to in locale format (12/21/2016).  now they are in ugly full GMT:
![image](https://cloud.githubusercontent.com/assets/908119/21283162/d6163f7a-c3b2-11e6-8f94-20d0c6001017.png)